### PR TITLE
atomically update scores

### LINKED ISSUE #46: Change our setScore operations for Firestore.increment
# Introduction

As we'll have now multiple `coverbot`s writing into our score database, we need to ensure we increase our node scores atomically, as otherwise if `N` bots write on a given node at the same time, instead of increasing their score by `relay_points * N` points, we'll be doing `relay_points` only.

# Task Description

- Change https://github.com/hoprnet/hopr-chatbot/blob/feature/basodino/src/bots/coverbot/index.ts#L116 to use [Firestore Increment](https://fireship.io/snippets/firestore-increment-tips/) instead. Make sure to use `score.Relayed` constant (seen used [here](https://github.com/hoprnet/hopr-chatbot/blob/feature/basodino/src/bots/coverbot/index.ts#L382)) to select the amount to increase.

### PR DESCRIPTION
**Pull request by nionis**
Fixes #46

### CODE REVIEW

**Comment by nionis** [comment]
unable to test this locally :(

File: src/bots/coverbot/index.ts [hunk 1]
```diff
@@ -443,11 +467,11 @@ export class Coverbot implements Bot {
...
 
       // 4.
-      const score = await this._getHoprAddressScore(relayerAddress)
-      const newScore = score + ScoreRewards.relayed
-
-      await Promise.all([
-        this._setHoprAddressScore(relayerAddress, newScore),
+      const [ newScore ] = await Promise.all([
+        this._increaseHoprAddressScore(relayerAddress, (prevScore) => {
+          if (!prevScore) return ScoreRewards.relayed
```
Comment (nionis) [inline src/bots/coverbot/index.ts:472]: here I decided to award the points even though the previous score should be `100`, my thinking is that awarding points shouldn't depend if the first `100` points were awarded

Comment (jjperezaguinaga) [reply src/bots/coverbot/index.ts:472]: Should be fine, let me go ahead and test it myself

**Review by jjperezaguinaga — APPROVED** [review]
LGTM, tested locally, all good.

**Pull request merged**